### PR TITLE
Tidied the slides

### PR DESCRIPTION
**Pull request by wxtim**
Fixed a few problems I noticed with the slides whilst checking #418 : Mostly slides where content was
- Too wide
- Too long
- Overwhelming

### CODE REVIEW

File: src/tutorial/runtime/introduction.rst [hunk 1]
```diff
@@ -236,17 +236,21 @@ The Cylc GUI
 Task & Job States
 ^^^^^^^^^^^^^^^^^
 
-- Task states have grey icons.
-- Job states have colour-coded squares.
+.. ifnotslides::
```
Comment (wxtim) [inline src/tutorial/runtime/introduction.rst:239]: Table didn't fit on the slide.

File: src/tutorial/runtime/introduction.rst [hunk 2]
```diff
@@ -372,9 +376,12 @@ Numbered run directories
    $ ls -l ~/cylc-run/my_workflow/runN
    ~/cylc-run/baz/runN -> run2
 
+.. nextslide::
```
Comment (wxtim) [inline src/tutorial/runtime/introduction.rst:379]: Existing slide was overwhelmingly busy IMO.

File: src/tutorial/runtime/runtime-configuration.rst [hunk 3]
```diff
@@ -65,7 +65,9 @@ Job Submission
    [runtime]
        [[hello_computehost]]
            script = echo "Hello Compute Host"
-           platform = powerful_computer  # Must be defined in Cylc global config!
+
+           # Must be defined in Cylc global config!
```
Comment (wxtim) [inline src/tutorial/runtime/runtime-configuration.rst:69]: Off the edge of th...

File: src/tutorial/scheduling/datetime-cycling.rst [hunk 4]
```diff
@@ -21,15 +21,15 @@ with numbered :term:`cycle points <cycle point>`.
    or every few hours. To support this Cylc can generate datetime sequences
    as :term:`cycle points <cycle point>` instead of integers.
 
-.. admonition:: Reminder
-   :class: tip
+   .. admonition:: Reminder
```
Comment (wxtim) [inline src/tutorial/scheduling/datetime-cycling.rst:24]: Too big for the slide, disappears off the bottom.

File: src/tutorial/scheduling/datetime-cycling.rst [hunk 5]
```diff
@@ -127,14 +127,17 @@ Inferred Recurrence
    front. For example, if the year is omitted then the recurrence can be
    inferred to be annual. E.g.:
 
-.. code-block:: sub
+.. csv-table::
```
Comment (wxtim) [inline src/tutorial/scheduling/datetime-cycling.rst:130]: I think that these are easier to read off the slide as a table.

File: src/tutorial/scheduling/datetime-cycling.rst [hunk 6]
```diff
@@ -178,20 +190,23 @@ might produce different results for the recurrences.
...
 .. list-table::
    :class: grid-table
+   :width: 50%
 
    * - .. code-block:: cylc
           :emphasize-lines: 2
 
           [scheduling]
-              initial cycle point = 2000-01-01T00
+              initial cycle point = \
```
Comment (wxtim) [inline src/tutorial/scheduling/datetime-cycling.rst:199]: I don't like this, but it stops the RH column being pushed off the edge of the slide.

File: src/tutorial/scheduling/graphing.rst [hunk 7]
```diff
@@ -281,7 +281,8 @@ Cylc Graphs
        [[graph]]
            R1 = """
                buy_ingredients => make_dough
-               pre_heat_oven & make_dough => bake_bread => sell_bread & clean_oven
+               pre_heat_oven & make_dough => bake_bread
```
Comment (wxtim) [inline src/tutorial/scheduling/graphing.rst:284]: Again,  to wide for the slide.

**Review by hjoliver — APPROVED** [review]
Ran through pretty quickly and haven't built the slides yet, but looks good :+1:

File: src/tutorial/runtime/runtime-configuration.rst (hunk 3 above)
Comment (oliver-sanders) [inline src/tutorial/runtime/runtime-configuration.rst:69]: Easier to get rid of this comment, it's an example, not expected to work.

```suggestion
```

File: src/tutorial/runtime/runtime-configuration.rst [hunk 8]
```diff
@@ -105,7 +107,9 @@ Job Submission
            # Submit to the host "big-computer".
            platform = slurm_platform
 
-           # Inform "slurm" that this job requires 500MB of RAM and 4 CPUs.
+           # Inform "slurm" that this job requires
+           # - 500MB of RAM
+           # - 4 CPUs
```
Comment (oliver-sanders) [inline src/tutorial/runtime/runtime-configuration.rst:112]:
```suggestion
           # job requires 500MB of RAM & 4 CPUs
```

**Review by oliver-sanders — APPROVED** [review]

File: src/tutorial/runtime/runtime-configuration.rst (hunk 3 above)
Comment (hjoliver) [reply src/tutorial/runtime/runtime-configuration.rst:69]: I think we should at least refer ahead in the text, with an aside e.g. "Platforms are explained later in ...".

**Pull request merged**